domain/domain.go

```diff
@@ -445,6 +445,7 @@ func (do *Domain) StatsHandle() *statistics.Handle {
 // should be called only once in BootstrapSession.
 func (do *Domain) LoadTableStatsLoop(ctx context.Context) error {
```

Comment:
This method does more that the method indicates now, should be renamed to contain the added work.